files/es/web/api/htmltableelement/align/index.md

```diff
@@ -16,7 +16,7 @@ varalignment =HTMLTableElement.align;
 
 ### Parámetros
 
-- `alignment` {{deprecatedGeneric('inline','HTML4')}}
+- `alignment` {{deprecated_inline('HTML4')}}
```

Comment:
When the param is provided, yari will still call deprecatedGeneric

https://github.com/mdn/yari/blob/b2b3b0c1b74b686b17f352b2b139584f161d90f4/kumascript/macros/Deprecated_Inline.ejs#L25-L29